tests/test-cases.js

```diff
@@ -1501,6 +1501,39 @@ describe('Parse Tests', function() {
 	}
 });
 
+var PARSE_ASYNC_ERROR_TESTS = [
+	{
+		description: "Simple download with int timeout timed out",
+		input: BASE_PATH + "verylong-sample.csv",
+		config: {
+			download: true,
+			downloadTimeout: 1
+		},
+		disabled: !XHR_ENABLED,
+		expected: {
+			error: "timeout"
```

Comment:
This doesn't look right compared to all the other test cases.

Shouldn't it be something like(?):
```javascript
expected: {
    data: [],
    errors: ["timeout"] // not even sure about this, there doesn't seem to be any other tests for exceptions
}
```

Reply:
No.